### DISCUSSION TITLE
[Esame 9 Gennaio 2024] Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#431] [Esami - Viaggi/Piazza] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/9f331f0b-86c7-411a-9267-3caa659dd6db)

**Comment by Elia-Belli** [comment, marked as answer]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/68f6218a-8d33-4025-a276-1b98db46b380)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/93085413-7474-42c8-be39-90247bad5e81)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/85cef166-ad30-44ee-8d2a-14559c079725)

**Comment by Frisbon** [comment]
![image_2024-01-28_10-51-37](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/e6d32486-c180-41d5-beaf-e3626cff8eb7)
